apps/admin/frontend/src/components/save_frontend_file_modal.tsx

```diff
@@ -233,31 +235,26 @@ export function SaveFrontendFileModal({
     throwIllegalValue(currentState);
   }
 
-  switch (usbDrive.status) {
-    case 'absent':
+  switch (usbDriveStatus.status) {
+    case 'no_drive':
     case 'ejected':
-    case 'bad_format':
+    case 'error':
       // When run not through kiosk mode let the user save the file
       // on the machine for internal debugging use
       return (
         <Modal
           title="No USB Drive Detected"
           content={
             <P>
-              <UsbImage
-                src="/assets/usb-drive.svg"
-                alt="Insert USB Image"
-                // hidden feature to save with file dialog by double-clicking
-                onDoubleClick={() => exportResults(true)}
```

Comment:
My opinion is that we shouldn't have "hidden features" in our open source code base. Even though they are, in that way, not hidden, it may be bad optics like calling mock functions "fake" functions. Also - the same capability is on this same modal anyway...